src/components/orgSelector.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from "react";
+import { useAllOrgs } from "dcs-react-hooks";
+import Autocomplete from "@mui/material/Autocomplete";
+import TextField from "@mui/material/TextField";
+
+export default function Organization() {
```

Comment:
I would add props here of: 
- organization
- onOrganization

These will be used instead of internal state. That way the value can be used by the rest of the app and not just internal to this component. We already store and set those in a hook for the app and pass that around where needed.